Resources/views/Sitemap/index.html.twig

```diff
@@ -1,7 +1,7 @@
 <ul class="cmf-sitemap">
     {% for url in urls %}
         <li>
-            <a href="{{ url.location }}" title="{{ url.label }}">{{ url.label }}</a>
+            {% if url.depth is not null %} <span class="indent-{{ url.depth }}"></span>{% endif %}<a href="{{ url.location }}" title="{{ url.label }}">{{ url.label }}</a>
```

Comment:
lets put the class on the <li>. the <ul> would be really difficult as we
would need to look-ahead somehow.